### ISSUE TITLE
An uploaded file doesn't show the Anvil! nodes to sync or not sync it to.

### ISSUE THREAD

**Issue by digimer** [ClusterLabs/anvil#443]
After uploading the file, I clicked on the edit icon to make sure it was sync'ing to the node. Nothing showed. Perhaps this was to soon after the upload and it hadn't been added to the files table yet? In any case, something useful should be displayed.

![image](https://github.com/ClusterLabs/anvil/assets/189162/2a323a26-bc69-4417-b681-0c41c5fd1726)

**Comment by digimer** [comment]
Changed to urgent and pre-beta; The file vanished, it's not in the /mnt/shared/incoming or files directories, and it's not in the database. Note that it _looked_ like the file uploaded, it took about the right amount of time. 

```
Aug 17 21:12:51 an-striker01.alteeve.com node[1295193]: Received GET /api/file
Aug 17 21:12:51 an-striker01.alteeve.com node[1295193]: Calling CLI script to get data.
Aug 17 21:12:51 an-striker01.alteeve.com node[1295193]: Access interact: {
Aug 17 21:12:51 an-striker01.alteeve.com node[1295193]:   "scriptId": "f5922f58-0b8f-47b9-aa92-362232d8749f",
Aug 17 21:12:51 an-striker01.alteeve.com node[1295193]:   "script": "f5922f58-0b8f-47b9-aa92-362232d8749f r  SELECT file_uuid, file_name, file_size, file_type, file_md5sum FROM files WHERE file_type != 'DELETED';\n"
Aug 17 21:12:51 an-striker01.alteeve.com node[1295193]: }
Aug 17 21:12:51 an-striker01.alteeve.com node[1295193]: Query stdout pre-hooks (type=[object]): []
Aug 17 21:12:51 an-striker01.alteeve.com node[1295193]: Query stdout post-hooks (type=[object]): []
Aug 17 21:12:51 an-striker01.alteeve.com node[1295193]: Touch session 6e5d877c-03ff-4700-b68d-d94e2e11b8bf: {
Aug 17 21:12:51 an-striker01.alteeve.com node[1295193]:   "session": {
Aug 17 21:12:51 an-striker01.alteeve.com node[1295193]:     "cookie": {
Aug 17 21:12:51 an-striker01.alteeve.com node[1295193]:       "originalMaxAge": 28800000,
Aug 17 21:12:51 an-striker01.alteeve.com node[1295193]:       "expires": "2023-08-18T09:12:51.672Z",
Aug 17 21:12:51 an-striker01.alteeve.com node[1295193]:       "httpOnly": true,
Aug 17 21:12:51 an-striker01.alteeve.com node[1295193]:       "path": "/"
Aug 17 21:12:51 an-striker01.alteeve.com node[1295193]:     },
Aug 17 21:12:51 an-striker01.alteeve.com node[1295193]:     "passport": {
Aug 17 21:12:51 an-striker01.alteeve.com node[1295193]:       "user": "b06c0644-226e-4574-b89c-5c6e83b595ec"
Aug 17 21:12:51 an-striker01.alteeve.com node[1295193]:     }
Aug 17 21:12:51 an-striker01.alteeve.com node[1295193]:   }
Aug 17 21:12:51 an-striker01.alteeve.com node[1295193]: }
Aug 17 21:12:51 an-striker01.alteeve.com node[1295193]: Access interact: {
Aug 17 21:12:51 an-striker01.alteeve.com node[1295193]:   "scriptId": "8196ae4f-9c0c-4fe5-9b15-e479ee192927",
Aug 17 21:12:51 an-striker01.alteeve.com node[1295193]:   "script": "8196ae4f-9c0c-4fe5-9b15-e479ee192927 w UPDATE sessions SET modified_date = '2023-08-17 21:12:51.676051225-04:00' WHERE session_uuid = '6e5d877c-03ff-4700-b68d-d94e2e11b8bf';\n"
Aug 17 21:12:51 an-striker01.alteeve.com node[1295193]: }
Aug 17 21:12:54 an-striker01.alteeve.com node[1295193]: Touch session 6e5d877c-03ff-4700-b68d-d94e2e11b8bf: {
Aug 17 21:12:54 an-striker01.alteeve.com node[1295193]:   "session": {
Aug 17 21:12:54 an-striker01.alteeve.com node[1295193]:     "cookie": {
Aug 17 21:12:54 an-striker01.alteeve.com node[1295193]:       "originalMaxAge": 28800000,
Aug 17 21:12:54 an-striker01.alteeve.com node[1295193]:       "expires": "2023-08-18T09:12:54.390Z",
Aug 17 21:12:54 an-striker01.alteeve.com node[1295193]:       "httpOnly": true,
Aug 17 21:12:54 an-striker01.alteeve.com node[1295193]:       "path": "/"
Aug 17 21:12:54 an-striker01.alteeve.com node[1295193]:     },
Aug 17 21:12:54 an-striker01.alteeve.com node[1295193]:     "passport": {
Aug 17 21:12:54 an-striker01.alteeve.com node[1295193]:       "user": "b06c0644-226e-4574-b89c-5c6e83b595ec"
Aug 17 21:12:54 an-striker01.alteeve.com node[1295193]:     }
Aug 17 21:12:54 an-striker01.alteeve.com node[1295193]:   }
Aug 17 21:12:54 an-striker01.alteeve.com node[1295193]: }
Aug 17 21:12:54 an-striker01.alteeve.com node[1295193]: Access interact: {
Aug 17 21:12:54 an-striker01.alteeve.com node[1295193]:   "scriptId": "02513a60-0583-40b3-a13e-89569121e8a4",
Aug 17 21:12:54 an-striker01.alteeve.com node[1295193]:   "script": "02513a60-0583-40b3-a13e-89569121e8a4 w UPDATE sessions SET modified_date = '2023-08-17 21:12:54.395606723-04:00' WHERE session_uuid = '6e5d877c-03ff-4700-b68d-d94e2e11b8bf';\n"
Aug 17 21:12:54 an-striker01.alteeve.com node[1295193]: }
Aug 17 21:12:54 an-striker01.alteeve.com node[1295193]: Error: EACCES: permission denied, open '/mnt/shared/incoming/Windows_Server_2022_eval_x64FRE_en-us.iso'
```

**Comment by digimer** [comment]
Permissions;

```
[root@an-striker01 ~]# ls -lah /mnt/shared/
total 0
drwxrwxr-x. 8 root root 99 Aug 12 15:47 .
drwxr-xr-x. 3 root root 20 Aug 12 15:47 ..
drwxrwxr-x. 2 root root  6 Aug 12 15:47 archives
drwxrwxr-x. 2 root root  6 Aug 12 15:47 definitions
drwxrwxr-x. 2 root root  6 Aug 12 15:47 files
drwxrwxr-x. 2 root root  6 Aug 12 15:47 incoming
drwxrwxr-x. 2 root root  6 Aug 12 15:47 provision
drwxrwxr-x. 2 root root  6 Aug 12 15:47 temp
```

**Comment by digimer** [comment]
Resolved in d255adc
